cognite_toolkit/_cdf_tk/loaders/_resource_loaders/location_loaders.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from collections.abc import Iterable
+from pathlib import Path
+from typing import final
+
+from cognite.client.data_classes.capabilities import Capability
+from cognite.client.utils.useful_types import SequenceNotStr
+
+from cognite_toolkit._cdf_tk.client.data_classes.locations import (
+    LocationFilter,
+    LocationFilterAcl,
+    LocationFilterList,
+    LocationFilterWrite,
+    LocationFilterWriteList,
+)
+from cognite_toolkit._cdf_tk.loaders._base_loaders import ResourceLoader
+from cognite_toolkit._cdf_tk.utils import CDFToolConfig, load_yaml_inject_variables
+
+
+@final
+class LocationFilterLoader(
+    ResourceLoader[str, LocationFilterWrite, LocationFilter, LocationFilterWriteList, LocationFilterList]
+):
+    folder_name = "locations"
+    filename_pattern = r"^.*LocationFilter$"
+    resource_cls = LocationFilter
+    resource_write_cls = LocationFilterWrite
+    list_cls = LocationFilterList
+    list_write_cls = LocationFilterWriteList
+    kind = "LocationFilter"
+    _doc_base_url = "https://api-docs.cogheim.net/redoc/#tag/"
+    _doc_url = "Location-Filters/operation/createLocationFilter"
+
+    @classmethod
+    def get_required_capability(cls, items: LocationFilterWriteList) -> Capability | list[Capability]:
+        if not items:
+            return []
+        return LocationFilterAcl(
+            actions=[LocationFilterAcl.Action.Read, LocationFilterAcl.Action.Write],
+            scope=LocationFilterAcl.Scope.All(),
+            allow_unknown=True,
+        )
+
+    @classmethod
+    def get_id(self, item: LocationFilter | LocationFilterWrite | dict) -> str:
+        if isinstance(item, dict):
+            return item["externalId"]
+        if not item.external_id:
+            raise KeyError("LocationFilter must have external_id")
+        return item.external_id
+
+    def load_resource(
+        self, filepath: Path, ToolGlobals: CDFToolConfig, skip_validation: bool
+    ) -> LocationFilterWrite | LocationFilterWriteList:
+        resources = load_yaml_inject_variables(filepath, {})
+        if isinstance(resources, dict):
+            return LocationFilterWrite.load(resources)
+        else:
+            return LocationFilterWriteList.load(resources)
```

Comment:
This is the same as the parent class, so no need to reimplement it.
```suggestion
```

However, you need to implement it and deal with all the `dataSetId` to `dataSetExternalId` replacements.